README.md

```diff
@@ -11,13 +11,38 @@ Supports
 * Gauges (including relative operations)
 * Sets
 
+Note: Only integers are supported for metric values.
+
 [![Build Status](https://secure.travis-ci.org/bitly/statsdaemon.png)](http://travis-ci.org/bitly/statsdaemon)
 
 Installing
 ==========
 
-```bash
-go get github.com/bitly/statsdaemon
+You can download either pre-compiled binaries or the source of the latest [release](https://github.com/bitly/statsdaemon/releases).
```

Comment:
For Installing can we list a 'Current Stable Release: vXXXX' section w/ links to the current release downloads below that, a block for older releases below that pointing to the github release page, and then have a `building from source` section similar to [nsq.io](http://nsq.io/deployment/installing.html)